LEAF_Request_Portal/sources/Signature.php

```diff
@@ -39,4 +39,46 @@ public function create($signature, $recordID, $message)
 
         return $this->db->getLastInsertID();
     }
+
+    public function getSignature($recordID)
+    {
+        $vars = array(
+            ':recordID' => $recordID,
+        );
+
+        $res = $this->db->prepared_query(
+            'SELECT * FROM 
+                signatures
+                WHERE recordID=:recordID;',
+            $vars
+        );
+
+        return $res;
+    }
+
+    public function getSignatureHistory($recordID)
+    {
+        $signatures = $this->getSignature($recordID);
+        $returnArray = array();
+        for ($i = 0; $i < count($signatures); $i++)
+        {
+            $vars = array(
+                ':recordID' => $recordID,
+                ':actionType' => 'sign',
```

Comment:
This isn't incorrect exactly, we just never talked about it. But when I worked on LEAF597 I added new action_types and the one I added for this was `signed`, not `sign`. This should be made consistent.